src/test/java/shop/RealItemTest.java

```diff
@@ -0,0 +1,20 @@
+package shop;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RealItemTest {
+
+    RealItem realItem = new RealItem();
+    double testWeight = 90.00;
+    double expectedTestWeight = 90.00;
+
+    @Test
+    @Tag("Items")
+    void testGetWeight() {
+        realItem.setWeight(testWeight);
+        assertEquals(expectedTestWeight, realItem.getWeight());
```

Comment:
it is not a good idea to verify getters and setters. For this app, for example, you can check, that virtual  item toString() method returns a string with expected `sizeOnDisk` value. The same for the real item